examples/browser/test/panels.spec.ts

```diff
@@ -8,6 +8,12 @@ describe('theia main elements loading', () => {
 
     before(() => {
         browser.url(url);
+        if (browser.getTitle() == 'localhost') {
+            browser.waitUntil(function () {
+            console.log('browser not loaded yet, trying again ');
+            return browser.getTitle() == '';
+            }, 3600000), 3000;        
+        }
```

Comment:
Sot it timeouts after 1h here ? that sounds way to much ? maybe 5 mins would be better ?
It could check faster too like 500ms for internval ?